src/orders/components/OrderCustomer/OrderCustomer.tsx

```diff
@@ -121,6 +110,15 @@ const OrderCustomer: React.FC<OrderCustomerProps> = props => {
     return "";
   };
 
+  const addressTextError = (orderError: OrderErrorFragment) => (
+    <>
+      <Typography variant="body2" className={classes.textError}>
+        {getOrderErrorMessage(orderError, intl)}
+      </Typography>
+      <FormSpacer />
+    </>
+  );
```

Comment:
Either make it component or copy it, component factories are antipattern in react